content/qamp/qamp-projects.json

```diff
@@ -0,0 +1,1232 @@
+    "name": "#26 Update and migrate Qiskit Terra fake backends"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Spring 23"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://github.com/repos/qiskit-advocate/qamp-spring-23/issues/27",
+    "mentees": [
+      {
+        "name": "Dimple Mevada"
+      },
+      {
+        "name": "Miika Vuorio"
+      },
+      {
+        "name": "Siddhartha Morales"
+      }
+    ],
+    "mentors": [
+      {
+        "name": "KUAN-CHENG C"
+      },
+      {
+        "name": "Tai Yue, Li"
+      }
+    ],
+    "name": "#27 Quantum Computing for Tomographic Image Reconstruction"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Fall 22"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://api.github.com/repos/qiskit-advocate/qamp-fall-22/issues/27",
+    "mentees": [],
+    "mentors": [
+      {
+        "name": "Ivan Carvalho"
+      }
+    ],
+    "name": "#27 ZX-Calculus pass for Qiskit Terra"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Spring 23"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://github.com/repos/qiskit-advocate/qamp-spring-23/issues/28",
+    "mentees": [
+      {
+        "name": "Inho Choi"
+      }
+    ],
+    "mentors": [
+      {
+        "name": "Ikko Hamamura"
+      }
+    ],
+    "name": "#28 Research on methods of simulation of Twirled Readout Error eXtinction (TREX) "
+  },
+  {
+    "cohort": {
+      "name": "QAMP Fall 22"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://api.github.com/repos/qiskit-advocate/qamp-fall-22/issues/28",
+    "mentees": [],
+    "mentors": [
+      {
+        "name": "Alberto Maldonado Romo"
+      }
+    ],
+    "name": "#28 tensor networks for QML"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Fall 22"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://api.github.com/repos/qiskit-advocate/qamp-fall-22/issues/29",
+    "mentees": [],
+    "mentors": [
+      {
+        "name": "James Weaver"
+      }
+    ],
+    "name": "#29 QNim"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Spring 23"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://github.com/repos/qiskit-advocate/qamp-spring-23/issues/30",
+    "mentees": [
+      {
+        "name": "Dimple Mevada"
+      },
+      {
+        "name": "Farai Mazhandu"
+      },
+      {
+        "name": "Mozammil Hassan"
+      }
+    ],
+    "mentors": [
+      {
+        "name": "Jayakumar Vaithiyashankar"
+      }
+    ],
+    "name": "#30 Developing Tutorials for Quantum Gates Analysis "
+  },
+  {
+    "cohort": {
+      "name": "QAMP Fall 22"
+    },
+    "collaborators": [],
+    "githubIssueUrl": "https://api.github.com/repos/qiskit-advocate/qamp-fall-22/issues/30",
+    "mentees": [],
+    "mentors": [
+      {
+        "name": "Matthew Treinish"
+      }
+    ],
+    "name": "#30 Improvements to Red Queen"
+  },
+  {
+    "cohort": {
+      "name": "QAMP Spring 23"
+    },
+    "collaborators": [
```

Comment:
I could be mistaken, but out of all 76 entries, this is the only project that has collaborators... is that strange to anyone else?

Reply:
It does seem weird, I agree 😬, but I did check and this is what is currently stored in Airtable as well. We can ask the team in charge of the Airtable base data to be sure.